src/main/java/mate/academy/service/ConnectionUtil.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.service;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://127.0.0.1:3306/test";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        String userName = "root";
+        String password = "Nero0410";
+        DB_PROPERTIES.put("user", userName);
+        DB_PROPERTIES.put("password", password);
+
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Can not load JDBC driver", e);
+        }
+    }
+
+    public static Connection getConnection() throws SQLException {
+        return DriverManager.getConnection(DB_URL,DB_PROPERTIES);
```

Comment:
The getConnection method should use try-with-resources or ensure that the Connection is closed properly elsewhere in the code to prevent potential resource leaks.